wiki/Ranking_Criteria/Difficulty_naming/en.md

```diff
@@ -45,7 +45,7 @@ These difficulty names are the most commonly used for each game mode.
 Mapsets of songs that originated from other rhythm games often borrow that game's difficulty naming scheme. Usage of these difficulty names is acceptable for any song. Some of the common naming schemes that made their way into osu! are listed here:
 
 ### Beatmania IIDX
-
+Cancel changes
```

Comment:
what's this about?

Reply:
I dont know

Reply:
If you don't want it, you can delete it separately.